packages/api/src/controllers/clip.ts

```diff
@@ -121,7 +121,13 @@ app.post(
       5,
     );
 
-    if (processingClips.length >= MAX_PROCESSING_CLIPS && !clippingUser.admin) {
+    // Hack to let StreamETH have a higher clipping limit without having to do the work to put this on the User object
+    let maxProcessingClips = MAX_PROCESSING_CLIPS;
+    if (req.user.id == "05f04681-b600-4a9b-81ea-1f56e7871f04") {
```

Comment:
WDYT of checking with `owner.id` instead, so it's applied to the owner of the content itself and not the user owning the api key doing the request? I am fine with it either way as only admins are allowed to do cross-clips, for now

Reply:
Yeah let's leave it for now then - we can hopefully roll this back after Devcon anyway